sdk/android/azure-android-compat/build.gradle.kts

```diff
@@ -0,0 +1,84 @@
+plugins {
+    id("com.android.application")
+}
+
+android {
+    namespace = "com.samples.androidcompat"
+    compileSdk = 33
+
+    defaultConfig {
+        applicationId = "com.samples.androidcompat"
+        minSdk = 26
+        targetSdk = 33
+        versionCode = 1
+        versionName = "1.0"
+
+        testInstrumentationRunner = "androidx.test.runner.AndroidJUnitRunner"
+    }
+
+    buildTypes {
+        release {
+            isMinifyEnabled = false
+            proguardFiles(
+                getDefaultProguardFile("proguard-android-optimize.txt"),
+                "proguard-rules.pro"
+            )
+        }
+    }
+    compileOptions {
+        sourceCompatibility = JavaVersion.VERSION_1_8
+        targetCompatibility = JavaVersion.VERSION_1_8
+    }
+
+    packagingOptions() {
+        resources.excludes.add("META-INF/*")
+    }
+}
+
+dependencies {
+    implementation("androidx.appcompat:appcompat:1.6.1")
+    implementation("com.google.android.material:material:1.9.0")
+    implementation("androidx.constraintlayout:constraintlayout:2.1.4")
```

Comment:
Do you have any UI in your sample? If not, you should look into whether these dependencies are actually needed.